catapult/plugins/apps.py

```diff
@@ -18,13 +18,38 @@
 import re
 
 from catapult.api import Plugin
+from catapult.api import PreferencesItem
 from catapult.api import SearchResult
 from catapult.i18n import _
 from gi.repository import Gio
+from gi.repository import Gtk
+
+
+class AppsPluginPrefs(PreferencesItem):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.label = Gtk.Label(label=_("Ignore OnlyShowIn Attr"))
+        self.widget = Gtk.Switch()
+        self.widget.connect("notify::active", self._on_widget_notify_active)
```

Comment:
I guess this is some testing leftover? Please remove this line and the empty callback method.

Reply:
Ok - attempting new push